### ISSUE TITLE
Add newline at end of `Syphon.h`

### ISSUE THREAD

**Issue by marcusstenbeck** [Syphon/Syphon-Framework#27]
Some compiler settings will cause warning or errors when there's no newline at the end of a file. Currently `Syphon.h` doesn't have a newline at EOF. Ignoring this is a bad idea, so I suggest adding this super simple fix.

```
/Users/theuser/Projects/capturetheweb/build/vendor/syphon/src/syphon-build/build/Debug/Syphon.framework/Headers/Syphon.h:184:4: error: 
      no newline at end of file [-Werror,-Wnewline-eof]
 */
   ^
1 error generated.
```
*My compiler settings that produce an error since `Syphon.h` doesn't have a newline at EOF.*

Thanks for this great project!

**Comment by bangnoise** [comment]
Thanks - fixed with 65ac2d3d768603fed5238086f14acae33f1e0909

**Comment by marcusstenbeck** [comment]
Perfect!
